framework/tests/spotify_ui_test.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from framework.page_forms.spotify_home_form import SpotifyHomeForm
+from framework.page_forms.artists_form import ArtistForm
+from framework.page_forms.search_song_form import SpotifySearchForm
+from framework.tests.base_test import prepare_browser_factory
+
+
+class TestSpotifyUI:
+    @pytest.fixture(scope="class")
+    def spotify_home_form(self, prepare_browser_factory):
+        return SpotifyHomeForm()
+
+    @pytest.fixture(scope="class")
+    def artist_form(self, prepare_browser_factory):
+        return ArtistForm()
+
+    @pytest.fixture(scope="class")
+    def search_song_form(self, prepare_browser_factory):
```

Comment:
I don't think you need a fixture for each page objects. You can define them as fields, as we don't search for any element at the moment of the form initialization:


    self._home_form = SpotifyHomeForm()
    
    
etc.